spec/text-editor-presenter-spec.coffee

```diff
@@ -20,6 +20,7 @@ describe "TextEditorPresenter", ->
 
       buffer = new TextBuffer(filePath: require.resolve('./fixtures/sample.js'))
       editor = atom.workspace.buildTextEditor({buffer})
+      editor.setUpdatedSynchronously(true)
```

Comment:
Ah, do you think we could drop it now then? I feel like the test is more realistic overall without it.